tsconfig.json

```diff
@@ -21,7 +17,5 @@
     "baseUrl": "src",
     "noFallthroughCasesInSwitch": true
   },
-  "include": [
-    "src"
-  ]
+  "include": ["src"]
```

Comment:
It looks like we have different prettier settings for json files. Can you revert this change on this file? Otherwise, we'll keep overwriting each other's changes.

Reply:
tsconfig formatting has been corrected